src/lib/Components/ArtworkGrids/Artwork.tsx

```diff
@@ -115,32 +115,47 @@ class Artwork extends React.Component<Props, any> {
     }
   }
 
-  saleMessage() {
-    const artwork = this.props.artwork
-    if (artwork.is_in_auction && artwork.sale_artwork) {
-      if (!artwork.sale_artwork.sale.is_closed) {
-        let numberOfBids = null
-        try {
-          numberOfBids = artwork.sale_artwork.bidder_positions_count
-        } catch (e) {
-          console.error(`Sentry issue #274707594 triggered with props: ${JSON.stringify(this.props)}`)
-          return null
-        }
-        let text = artwork.sale_artwork.opening_bid.display
-        if (numberOfBids > 0) {
-          text = `${artwork.sale_artwork.current_bid.display} (${numberOfBids} bid${numberOfBids === 1 ? "" : "s"})`
-        }
-        return (
-          <View style={{ flexDirection: "row" }}>
-            <Image style={{ marginRight: 4 }} source={require("../../../../images/paddle.png")} />
-            <SerifText style={styles.text}>{text}</SerifText>
-          </View>
-        )
-      } else {
-        return <SerifText style={styles.text}>Auction Closed</SerifText>
-      }
-    } else {
-      return artwork.sale_message && <SerifText style={styles.text}>{artwork.sale_message}</SerifText>
+  saleInfoLine() {
+    const { artwork } = this.props
+    const { sale } = artwork
+    const inClosedAuction = sale && sale.is_auction && sale.is_closed
+
+    return (
+      <Flex flexDirection={"row"} mb={2}>
+        <Sans color={color("black100")} weight={"medium"} size={"2"}>
+          {inClosedAuction ? "Bidding closed" : this.saleMessageOrBidInfo()}{" "}
+        </Sans>
+        <Sans size={"2"} color={color("black60")}>
```

Comment:
Note that for strings you don't need the `{}` brackets. Also, can you try removing `color('black60')` and just passing in `black60`? This should be fixed.

Reply:
<img width="285" alt="screen shot 2018-09-27 at 4 16 00 pm" src="https://user-images.githubusercontent.com/1457859/46172144-a7138480-c270-11e8-853f-57b02f9e3b4d.png">

At least in development, this seems to still be an issue for me, as well as in Reaction for `Sans` tags. It outputs `color: "black60"` in the generated CSS.

Reply:
I believe the issue is that the component tree isn't wrapped in a `<Theme>` component from Palette. I'm not sure exactly where the correct placement of this would be in this specific tree, but in the meantime would you mind adding a `// TODO: Look into wrapping in <Theme> component to remove `color` util functions` (or something) somewhere in this file?

Reply:
I think the end goal should be to have it up at the top of Home/Gene/Artist/Favourites etc